docs/getting-started-guides/kubeadm.md

```diff
@@ -0,0 +1,245 @@
+To learn more about the sample microservices app, see the [GitHub README](https://github.com/microservices-demo/microservices-demo).
+
+    # git clone https://github.com/microservices-demo/microservices-demo
+    # kubectl apply -f microservices-demo/deploy/kubernetes/manifests
+
+You can then find out the port that the [NodePort feature of services](/docs/user-guide/services/) allocated for the front-end service by running:
+
+    # kubectl describe svc front-end
+    Name:                   front-end
+    Namespace:              default
+    Labels:                 name=front-end
+    Selector:               name=front-end
+    Type:                   NodePort
+    IP:                     100.66.88.176
+    Port:                   <unset> 80/TCP
+    NodePort:               <unset> 31869/TCP
+    Endpoints:              <none>
+    Session Affinity:       None
+
+It takes several minutes to download and start all the containers, watch the output of `kubectl get pods` to see when they're all up and running.
+
+Then go to the IP address of your cluster's master node in your browser, and specify the given port.
+So for example, `http://<master_ip>:<port>`.
+In the example above, this was `31869`, but it is a different port for you.
+
+If there is a firewall, make sure it exposes this port to the internet before you try to access it.
+
+### Explore other add-ons
+
+See the [list of add-ons](/docs/admin/addons/) to explore other add-ons, including tools for logging, monitoring, network policy, visualization &amp; control of your Kubernetes cluster.
+
+
+## What's next
+
+* Learn more about [Kubernetes concepts and kubectl in Kubernetes 101](/docs/user-guide/walkthrough/).
+* Install Kubernetes with [a cloud provider configurations](/docs/getting-started-guides/) to add Load Balancer and Persistent Volume support.
+
+
+## Cleanup
+
+* To uninstall the socks shop, run `kubectl delete -f microservices-demo/deploy/kubernetes/manifests` on the master.
+* To uninstall Kubernetes, simply delete the machines you created for this tutorial.
+  Or alternatively, uninstall the `kubelet`, `kubeadm` and `kubectl` packages and then manually delete all the Docker container that were created by this process.
+
+## Feedback
+
+* Slack Channel: [#sig-cluster-lifecycle](https://kubernetes.slack.com/messages/sig-cluster-lifecycle/)
+* Mailing List: [kubernetes-sig-cluster-lifecycle](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)
+* [GitHub Issues](https://github.com/kubernetes/kubernetes/issues): please tag `kubeadm` issues with `@kubernetes/sig-cluster-lifecycle`
+
+## Limitations
+
+Please note: `kubeadm` is a work in progress and these limitations will be addressed in due course.
+
+1. The cluster created here doesn't have cloud-provider integrations, so for example won't work with (for example) [Load Balancers](/docs/user-guide/load-balancer/) (LBs) or [Persistent Volumes](/docs/user-guide/persistent-volumes/walkthrough/) (PVs).
```

Comment:
It doesn't _by default_, but refer to the `kubeadm reference` doc if you want to do enable cloud provider integrations